js/src/ui/ParityBackground/parityBackground.js

```diff
@@ -16,26 +16,17 @@
 
 import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
-import { bindActionCreators } from 'redux';
 
 class ParityBackground extends Component {
-  static contextTypes = {
-    muiTheme: PropTypes.object.isRequired
-  }
-
   static propTypes = {
+    style: PropTypes.object.isRequired,
```

Comment:
> So we'd have to choose whether to do it in the component or have a state-less component with props managed by Redux.

A general note:

Both Redux & Mobx encourage users to split React components into "dumb"/presentational and "smart"/behavioural. We'd have a smart component connecting the store/state to the React components tree and lots of "dumb", pure components beneath.

However, memoizing the background computation and returning a value that can be checked for equality by React should work in terms of performance.